.github/workflows/test-docs.yml

```
@@ -0,0 +1,25 @@
+---
+# This workflow sets the Test / test status check to success in case it's a docs only PR and test.yml is not triggered
+# https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/defining-the-mergeability-of-pull-requests/troubleshooting-required-status-checks#handling-skipped-but-required-checks
+name: Test # The name must be the same as in test.yml
+
+on:
+  push:
+    branches:
+      - main
+    paths-ignore: # This expression needs to match the paths ignored on test.yml.
+      - '**'
+      - '!**/*.md'
+  pull_request:
+    paths-ignore: # This expression needs to match the paths ignored on test.yml.
+      - '**'
+      - '!**/*.md'
```

```suggestion
    paths: # This expression needs to match the paths ignored on test.yml.
      - '**.md'
```
We can use the `paths` expression instead of the `paths-ignore` expression.

---

I don't recall what's the reason we used `paths-ignore` but that's how we have been doing in other places:
- https://github.com/elastic/apm-agent-java/blob/5ff1430a37021be595aca1777914cfb2c3f15a04/.github/workflows/main-docs.yml#L7-L10

I'd say to keep the same functionality for now